cli/internal/cmd/configfetchmeasurements.go

```diff
@@ -146,19 +146,19 @@ func (cfm *configFetchMeasurementsCmd) parseFetchMeasurementsFlags(cmd *cobra.Co
 	if err != nil {
 		return &fetchMeasurementsFlags{}, err
 	}
-	cfm.log.Debugf("Parsed measurements URL")
+	cfm.log.Debugf("Parsed measurements URL as %q", measurementsURL.String())
```

Comment:
measurementsURL can be nil, that's why the unit test fails. We also already log this in parseURLFlag(), so it think we can just delete these two lines.

Reply:
Changed this to %v. Would opt to keep this generally since it is a parsed output, thus can be useful for debugging.